### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by NamesJoeyWheeler** [ezelkow1/discord-key-bot#40]
I am trying to run this, but I get this error:
```
Db File does not exist, creating
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x40 pc=0x67d7e3]

goroutine 1 [running]:
main.Load(0x0, 0x0, 0x6a9580, 0x8bf530)
        E:/My Documents/GitHub/discord-key-bot/utils.go:102 +0xa3
main.main()
        E:/My Documents/GitHub/discord-key-bot/main.go:112 +0x273
```
My E: is a Blu-Ray drive and doesn't even have these files.
I have filled out the Token and BroadcastChannel 
Have I done something incorrectly?

**Comment by ezelkow1** [comment]
Did you also set a filename in the conf.json for "DbFile"?

If you just give a filename only it should try to make that in the local dir where the bot is being run from.  Those errors I believe are a permissions issue, I can see them on my box if for example I set the dbfile to be "c:/test", I will get those errors unless I run the bot as administrator.  So whatever path you have setup make sure the bot has permissions to store the dbfile there

I believe the E:/ is because when I built for windows that is where those files existed, so its just a reference from the build on my machine and not actually trying to use anything from your machine in E:
